app/homepage/index.js

```diff
@@ -47,6 +47,22 @@ const addNumberOfEarthDays = () => {
 		.append(`${Math.round(earthDaysSinceLanding)} Earth days`);
 };
 
+const getPhotoOfTheDay = () => {
+	fetch('http://localhost:3000/apod')
```

Comment:
Is it possible to make this URL relative so that it'll work even if we were to, say, deploy it?

Reply:
Yep, swapped to a relative URL.